packages/ag-charts-community/src/chart/mapping/prepare.ts

```diff
@@ -350,6 +354,19 @@ function prepareAxis<T extends AxesOptionsTypes>(
         axis.crossLines = axis.crossLines.map((crossLine) => jsonMerge([crossLinesTheme, crossLine]));
     }
 
+    if (axis.gridLine !== undefined) {
+        const { style } = axis.gridLine;
+        if (Array.isArray(style)) {
+            axis.gridLine.style = style.map((value) => {
+                const { stroke, lineDash } = value;
+                if (stroke === undefined && lineDash !== undefined) {
+                    return { stroke: defaultGridStroke, lineDash };
+                }
+                return value;
+            });
+        }
```

Comment:
Would it be possible to take a similar approach to `crossLines` above (i.e. JSON merge of array elements with theme options), rather than treating `gridLine.style` as an exception that is passed in as a parameter?